authors/apps/authentication/urls.py

```diff
@@ -5,7 +5,7 @@
 )
 
 urlpatterns = [
-    path('user/', UserRetrieveUpdateAPIView.as_view()),
-    path('users/', RegistrationAPIView.as_view()),
-    path('users/login/', LoginAPIView.as_view()),
+    path('user/', UserRetrieveUpdateAPIView.as_view(), name="detail"),
+    path('users/', RegistrationAPIView.as_view(), name="users"),
```

Comment:
@fahadmak Please change this to more descriptive name
```suggestion
    path('users/', RegistrationAPIView.as_view(), name="register"),
```

Reply:
Let me know when you are done with this so i can merge in this.

Reply:
i am done with the changes